tests/unit/version2/issueSearch.test.ts

```diff
@@ -75,7 +75,9 @@ test('searchForIssuesUsingJqlEnhancedSearch should calls without parameters', ({
   const client = new Version2Client(config);
   const sendRequestStub = sinon.stub(client, 'sendRequest');
 
-  client.issueSearch.searchForIssuesUsingJqlEnhancedSearch({});
+  client.issueSearch.searchForIssuesUsingJqlEnhancedSearch({
```

Comment:
This change I really wasn't sure, it sounded like the test want to be able to run without any parameter, but `jql` is a required parameter. 
I just fixed the build, but I'm not 100% sure what is the real issue here

Reply:
Thanks for catching this - I believe it was overlooked during development